arch/sim/src/sim/posix/sim_x11eventloop.c

```diff
@@ -105,7 +105,7 @@ void sim_x11events(void)
 
   /* Check if there are any pending, queue X11 events. */
```

Comment:
```suggestion
  /* Dequeue any pending X11 events */
```
Update the documentation for the changed line.

Reply:
done